src/Toggle/IsToggleActive.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace Clearbooks\Labs\Toggle;
+
+use Clearbooks\Labs\Toggle\Gateway\ActivatableToggleGateway;
+
+class IsToggleActive implements \Clearbooks\Labs\Toggle\UseCase\IsToggleActive
+{
+    /**
+     * @var ActivatableToggleGateway
+     */
+    private $gateway;
+
+    /**
+     * @param ActivatableToggleGateway $gateway
+     */
+    public function __construct( ActivatableToggleGateway $gateway )
+    {
+        $this->gateway = $gateway;
+    }
+
+    /**
+     * @param string $toggleName
+     * @return bool is it active
+     */
+    public function isToggleActive( $toggleName )
+    {
+        $toggle = $this->gateway->getActivatableToggleByName( $toggleName );
+        return $toggle && $toggle->isActive() ?: false;
```

Comment:
This interactor and its logic supposed to decide if a toggle is active. Not some gateway or entity!

My suggestions are:
* Is a toggle exists and in the current Public Release or Stable Release
* And visible (not existing or unknown toggles not supposed to be visible)
* Current group admins not disabled it 
* Current user enabled it (LABS-7)

@Clearbooks-Tom @craigjbass